src/main/java/com/openshift/internal/restclient/model/volume/NFSVolume.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.internal.restclient.model.volume;
+
+import java.util.Objects;
+
+
+public class NFSVolume {
```

Comment:
Would expect to implement an INFSVolume interface.  We have discussed modifying the backing JSON library and this would mitigate consumers of carrying about this specific implementation